.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "import"],
   rules: {
+    "@typescript-eslint/ban-ts-comment": "warn",
```

Comment:
Just satisfying my compulsive need to alphabetize here :)

Reply:
What is the idea with warning rules?